app/src/main/java/org/tuerantuer/launcher/ui/MainActivity.kt

```diff
@@ -121,6 +155,12 @@ fun LauncherApp(
     screenTransitionManager: ScreenTransitionManager,
     uiState: UiState,
 ) {
+    val backgroundColor = when {
+        uiState.settings.wallpaperType != WallpaperType.SOLID_COLOR
+                && uiState.screenState is ScreenState.HomeScreenState
+        -> MaterialTheme.colorScheme.background.copy(alpha = 0f)
```

Comment:
`MaterialTheme.colorScheme.background.copy(alpha = 0f)`
I would write Color.Transparent again for clarity